src/client/src/apps/MainRoute/layouts/reducer.ts

```diff
@@ -46,7 +46,7 @@ export const layoutReducer = (
             ...state,
             spotTiles: {
               ...state.spotTiles,
-              [action.payload.name]: getWindowPosition(action),
+              [action.payload.name!]: getWindowPosition(action),
```

Comment:
I would not write it in a new code, but here it's a quick win for existing code fix